### PR TITLE
Updated some comments. Added the AgentAvailability.message field.

### PR DESCRIPTION
**Pull request by TazmanianDI**

### CODE REVIEW

File: src/common/types/serviceDeskCallback.ts [hunk 1]
```diff
@@ -91,22 +91,25 @@ interface ServiceDeskCallback {
...
 interface AgentAvailability {
   /**
-   * Number of users ahead in the queue.
+   * The current position of the user in a queue. E.g. "You are number 2 in line."
    */
   position_in_queue?: number;
 
   /**
-   * Estimated wait time in minutes.
+   * The estimated wait time for the user in minutes. E.g. "Current wait times is 2 minutes."
```
Comment (jeesooxkim) [inline src/common/types/serviceDeskCallback.ts:103]:
```suggestion
   * The estimated wait time for the user in minutes. E.g. "Current wait time is 2 minutes."
```

**Review by jeesooxkim — CHANGES_REQUESTED** [review]
Just added a typo fix. Will approve after!

**Review by jeesooxkim — APPROVED** [review]
LGTM!

**Review by jeesookim-ibm — APPROVED** [review]
LGTM

**Pull request merged**